tools/BUILD

```diff
@@ -22,6 +27,40 @@ _SCHEME_DIAGNOSTICS = xcode_schemes.diagnostics(
 )
 
 _SCHEMES = [
+    xcode_schemes.scheme(
+        name = "bazel_dependencies",
+        launch_action = xcode_schemes.launch_action(
+            _TOOLS["bazel_dependencies"],
+            args = [
+                # outputPath
+                "/tmp/pbxproj_partials/bazel_dependencies",
+                # targetIdsFile
+                "bazel-out/darwin_arm64-dbg/bin/external/_main~internal~rules_xcodeproj_generated/generator/tools/xcodeproj/xcodeproj_target_ids",
+                # indexImport
+                "bazel-out/darwin_arm64-opt-exec-2B5CBBC6/bin/external/_main~non_module_deps~rules_xcodeproj_index_import/index-import",
+                # xcodeConfigurations
+                "--xcode-configurations",
+                "Debug",
+                "Release",
+                # defaultXcodeConfiguration
+                "--default-xcode-configuration",
+                "Release",
+                # platforms
+                "--platforms",
+                "macosx",
+                "iphonesimulator",
+                # preBuildScript
+                # postBuildScript
```

Comment:
I assume this is not a leftover right? It's here to show the reader that these options are also available

Reply:
Yeah